doc/releases/migration-guide-4.0.rst

```diff
@@ -288,5 +288,14 @@ Shell
 * ``kernel threads`` and ``kernel stacks`` shell command have been renamed to
   ``kernel thread list`` & ``kernel thread stacks``
 
+JWT (JSON Web Token)
+====================
+
+* By default signature is now computed through PSA Crypto API for both RSA and ECDSA.
```

Comment:
I think you're missing a word here... `By default, <something> signature is now... `

Reply:
Rather than a word I would say an article, "the". JWT assumes that the JSON content is signed and "signature" is the subject of this sentence, so I would say that "by default, the signature" should be fine.
However I'm not a native english speaker, so I might miss something so please I apologize for any mistake.